helm/template-operator/templates/_helpers.tpl

```diff
@@ -19,7 +19,7 @@ Common labels
 {{- define "labels.common" -}}
 app: {{ include "name" . | quote }}
 {{ include "labels.selector" . }}
-app.giantswarm.io/branch: {{ .Values.project.branch | quote }}
+app.giantswarm.io/branch: {{ .Values.project.branch | replace "#" "-" | quote }}
```

Comment:
I think we also need to truncate the branch name for dependabot PRs. This is what we're using in app-operator but I prefer what you're using for Release.Name.

https://github.com/giantswarm/app-operator/blob/9923dccd1a963bda59556807ec371376fcfe6016/helm/app-operator/templates/_helpers.tpl#L22

Reply:
Replacing more characters now, PTAL.